src/module/collection/actors.ts

```diff
@@ -1,7 +1,13 @@
-import { ActorPF2e } from "@actor";
+import { ActorPF2e, PartyPF2e } from "@actor";
 import { ActorUpdateContext } from "@actor/base.ts";
 
 export class ActorsPF2e<TActor extends ActorPF2e<null>> extends Actors<TActor> {
+    /** The world's active party, if one exists */
+    get party(): PartyPF2e<null> | null {
+        // Eventually we'll need to implement active parties, but this is stubbed for development of other features
+        return (this as Actors<ActorPF2e<null>>).find<PartyPF2e<null>>((a) => a.isOfType("party")) ?? null;
+    }
```

Comment:
Why an actor instance getter that doesn't check whether it's the party the actor is in?

Reply:
Its not an actor instance getter, its a game.actors instance getter. As per the description.